modules/fb_instant_articles_display/src/DrupalInstantArticleDisplay.php

```diff
@@ -28,8 +30,11 @@
 /**
  * Facebook Instant Article node wrapper class.  Builds up an InstantArticle
  * object using field formatters.
+ *
+ * Class DrupalInstantArticleDisplay
+ * @package Drupal\fb_instant_articles_display
  */
-class InstantArticleNodeWrapper {
+class DrupalInstantArticleDisplay {
```

Comment:
Now that this class is namespaces under `Drupal` it seems redundant to have that also in the name. I would suggest maybe naming something that is a little more descriptive. To me, this feels like an _Adapter_ more than a _Wrapper_ or a _Display_. Essentially it adapts/transforms a Drupal node to an InstantArticle, right? So we could consider calling it something like `Drupal\fb_instant_articles_display\NodeAdapter\`